README.md

```diff
@@ -1,25 +1,78 @@
 # catalyst-slack-service
 
-A microservice that integrates Slack and Bias Correction Service. This service subscribes to message events on Slack and utilizes the Bias Correction Service for potential suggestions to replace biased terms.
+A microservice that integrates Slack and [Bias Correction Service](https://github.com/willowtreeapps/catalyst-bias-correct-service). This service subscribes to message events on Slack and utilizes the Bias Correction Service for potential suggestions to replace biased terms.
 
-## Setup
+## Prerequisites
+* [JDK 12](https://www.oracle.com/java/technologies/javase/jdk12-archive-downloads.html)
+* [sbt](https://www.scala-sbt.org/download.html)
+* [docker](https://hub.docker.com/editions/community/docker-ce-desktop-mac)
+* [redis](https://redis.io/topics/quickstart) 
+    - ```docker run --name my-redis -p 6379:6379 --restart always --detach redis```
+* TODO - Slack Permissions
 
-Download IntelliJ IDEA
-Open this folder as a project in IDEA
-Allow IDEA to download/configure the project as necessary
-Navigate to the sbt shell pane and allow IDEA to configure it as necessary
-When the shell is ready, type run
-Navigate to http://localhost:4542 to verify that the service is running
+## Run
 
-## Packaging as a Docker container
+You can run this project using IntelliJ IDEA or Docker. You will need the same configuration variables for both.
 
+```
+REDIS_HOST=localhost
+REDIS_PORT=6379
+PLAY_SECRET_KEY={TODO}
```

Comment:
just making sure the TODO here is intended :)

Reply:
Good catch.